domains/battlehawk.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "is-a.dev domain for BATTLEHAWK00(redirecting to https://code1024.icu)",
+  "owner": {
+    "username": "BATTLEHAWK00",
+    "email": "yxlk2oi@gmail.com"
+  },
+  "record": {
+    "URL": "https://code1024.icu"
+  },
+  "proxied": true
```

URL records are proxied by default
```suggestion
    "URL": "https://code1024.icu"
  }
```